### ISSUE TITLE
Typo "aria-model"

### ISSUE THREAD

**Issue by jfhector** [accessibilitysupported/a11ysupport.io#95]
On the page ["Test: APG 1.1 modal dialog example"](https://a11ysupport.io/tests/apg__modal-dialog-example), one of the assertion reads:
> "MUST convey the presence of aria-model=true"

`aria-model` should be `aria-modal`
